### DISCUSSION TITLE
themes folder

### DISCUSSION THREAD

**Discussion by hangangmagazine** [hugo-themes/toha#337] [Q&A] [answered]
Appreciate the amazing theme.

I notice that I have this themes/toha/ folder that no one else seems to have.  

But when I delete this folder, my site can no longer run locally. 
I tried to copy paste the theme/toha contents into their corresponding folders in the root directory and then delete the theme/ folder but it still returned an error when I tried to do `hugo server`

I'm sure this is obvious but I'm not sure what's going on.

**Comment by hossainemruz** [comment, marked as answer]
The theme should be added as git submodules in `themes/toha` folder. So, the folder is expected to be there.

**Reply by hangangmagazine** [reply]
Oh okay, I won't touch it again then. Thank you so much
